qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/modules/kubernetes/services/clearml/variables.tf

```diff
@@ -24,5 +24,4 @@ variable "external-url" {
 
 variable "tls" {
   description = "TLS configuration"
-  type        = object({})
```

Comment:
Should there be a different type specified then?

Reply:
The problem here is you need to have explicit keys for the object, you can't have arbitrary keys (like say `**kwargs` in python) and even the optional keys in the typing is an experimental feature in terraform.

Reply:
and we don't know the exact keys beforehand:

https://github.com/Quansight/qhub/blob/e3eb1d309714f4d53a620b9e4ec453462c7822b9/qhub/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D/infrastructure/modules/kubernetes/services/meta/qhub/locals.tf#L2-L6